src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -639,6 +645,25 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
                 newNode
             ]);
             setElements(positionedElements);
+
+            // Center pane focus on the new node
+            const positionedX =
+                positionedElements[positionedElements.length - 1].position.x;
+            const positionedY =
+                positionedElements[positionedElements.length - 1].position.y;
+
+            const wrapperBoundingBox = reactFlowWrapperRef.current.getBoundingClientRect();
+
+            rfInstance.setTransform({
+                x:
+                    -positionedX * currentLocation.zoom +
+                    wrapperBoundingBox.width / 2 -
+                    nodeWidth / 2,
+                y:
+                    -positionedY * currentLocation.zoom +
+                    wrapperBoundingBox.height / 2,
```

Comment:
So `x` considers the `nodeWidth` but `y` doesn’t? Shouldn’t it consider `nodeHeight / 2`?